app/javascript/components/editor/TestsGroupedByStatusList.tsx

```diff
@@ -14,9 +14,9 @@ const Tests = ({
 }): JSX.Element => {
   return (
     <>
-      {tests.map((test) => (
+      {tests.map((test, i) => (
         <TestSummary
-          key={test.name}
+          key={i}
```

Comment:
Is this because test names are not guaranteed to be unique?

Reply:
Yes. We have no control over them, so it is probably unwise to use it as a key